lib/doc_builder_testing/doc_builder_wrapper.rb

```diff
@@ -25,15 +25,16 @@ def default_builder_location
   end
 
   # @return [String] command to run builder for any platform
-  def run_build_command(script_file)
-    "#{@builder_exe} #{script_file} 2>&1"
+  def run_build_command(script_file, **arguments)
```

Comment:
let's call them not argument but `kwargs` - this is more common name for list of keyword argument

Also please change docstring of the method, maybe even with simple example so we know how to use it